dialer.go

```diff
@@ -159,7 +163,12 @@ func NewDialer(ctx context.Context, opts ...DialerOption) (*Dialer, error) {
 		dialerID:       uuid.New().String(),
 		iamTokenSource: cfg.tokenSource,
 		dialFunc:       cfg.dialFunc,
+		metrics:        mc,
+		openConns:      make(map[string]int64),
```

Comment:
Would it be better for openConns to be a property on cloudsql.Instance?

Reply:
Done.